samples/snippets/webhook_prebuilt_telecom.py

```diff
@@ -0,0 +1,253 @@
+# Copyright 2021, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" handle_webhook will return the correct fullfilment response dependong the tag that is sent in the request"""
+# [START dialogflow_cx_v3_webhook_prebuilt_telecom]
+import copy
+
+
+def cxPrebuiltAgentsTelecom(request):
+    print('Cloud Function:' + 'Invoked cloud function from Dialogflow')
```

Comment:
Can you change over to the logging module, and log this at "INFO" level? default logger should be fine. This is best practice when the code will be invoked on a server.